packages/x-charts/src/BarChart/BarPlot.tsx

```diff
@@ -218,12 +218,6 @@ const BarPlotRoot = styled('g', {
   slot: 'Root',
   overridesResolver: (_, styles) => styles.root,
 })({
-  [`& .${barElementClasses.highlighted}`]: {
-    filter: 'brightness(120%)',
-  },
-  [`& .${barElementClasses.faded}`]: {
-    opacity: 0.3,
-  },
   [`& .${barElementClasses.root}`]: {
     transition: 'opacity 0.2s ease-in, fill 0.2s ease-in',
   },
```

Comment:
😈 do we even need transition?

Reply:
I know it's a joke, but her is the impact of removing the transition. It adds a bit of smoothness when moving the mouse :)

https://github.com/user-attachments/assets/b26eb4b7-1991-464e-b91f-aacb8d3382c7